src/ViewLayer/ViewContainer.h

```diff
@@ -39,27 +37,32 @@ class ViewContainer
     ~ViewContainer();
 
 private:
-    I_DisplayDashboardUI* DisplayDashboardUI_;
-    I_RaceModeDashboardUI* RaceModeDashboardUI_;
-    BatteryUi* batteryUi_;
-    ControlUi* controlUi_;
-    HomePageUi* homepageUi_;
-    FaultUi* faultUi_;
-    MotorUi* motorUi_;
-    MpptUi* mpptUi_;
-    TabUi* tabUi_;
-    ProgressBar* ProgressBar_;
-
-    QScopedPointer<MotorView> MotorView_;
-    QScopedPointer<FaultView> FaultView_;
-    QScopedPointer<ControlView> ControlView_;
-    QScopedPointer<MpptView> MpptView_;
-    QScopedPointer<HomePageView> HomePageView_;
+    QScopedPointer<I_DisplayDashboardUI> displayDashboardUI_;
+    QScopedPointer<I_RaceModeDashboardUI> raceModeDashboardUI_;
+    QScopedPointer<I_BatteryUi> batteryUi_;
+    QScopedPointer<I_ControlUi> controlUi_;
+    QScopedPointer<I_HomePageUi> homepageUi_;
+    QScopedPointer<I_FaultUi> faultUi_;
+    QScopedPointer<I_MotorUi> motorUi_;
+    QScopedPointer<I_MpptUi> mpptUi_;
+    QScopedPointer<I_TabUi> tabUi_;
+    QScopedPointer<I_AuxBMSUi> auxBMSUi_;
+    QScopedPointer<ProgressBar> progressBar_;
 
+    QScopedPointer<MotorView> motorView_;
+    QScopedPointer<FaultView> faultView_;
+    QScopedPointer<ControlView> controlView_;
+    QScopedPointer<MpptView> mpptView_;
+    QScopedPointer<HomePageView> homePageView_;
+    QScopedPointer<AuxBMSView> auxBMSView_;
 
     QScopedPointer<DisplayDashboardView> DisplayDashboardView_;
```

Comment:
You renamed `raceModeDashboardView_`, might as well rename `DisplayDashboardView_` to be consistent